p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/EnterSeedViewController/EnterSeedMainButtonContent.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  EnterSeedMainButtonContent.swift
+//  p2p_wallet
+//
+//  Created by Andrew Vasiliev on 18.11.2021.
+//
+
+enum EnterSeedMainButtonContent {
```

Comment:
Enum that releated to a scene should be defined inside that scene.
Let's add this enum MainButtonContent inside EnterSeed.swift, after NavigatableScene, the same for InvalidationReason.
Avoid polluting global namespace